alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+
+let answer = "";
+
+let numberOfBuildings = streetArray.length;
+let tallestBuildingSoFar = 0;
+let tallestBuilding = Math.max(...streetArray);
+
+let bannerOneHeight = tallestBuilding;
+let bannerOneWidth = numberOfBuildings;
+let bannerOneArea = tallestBuilding * numberOfBuildings;
+let bannerTwoHeight = tallestBuilding;
+let bannerTwoWidth = numberOfBuildings;
+let bannerTwoArea = tallestBuilding * numberOfBuildings;

```

Comment:
Should we start off with two banners initialized to cover the max area? I would've expected only one

Reply:
You know, I guess it really doesn't matter.
Worst possible case though is still one banner covering everything 🤷